guidelines/content-guidelines/04-formatting.md

```diff
@@ -126,6 +126,7 @@ Ideally, headings fit into one line in the generated output. Be concise, but als
 * Use action verbs and present tense verbs in headings when possible, especially in tutorials. For example, **Add a document type**.
 * While gerunds are acceptable in body-level content, DO NOT use gerunds in headings. Use **Create a storefront** instead of **Creating a storefront**.
 * Avoid stacked headings, which are headings without body-level content in between. For example, DO NOT use a Heading 2 (H2) to introduce one or more Heading 3s. Instead, add a paragraph after the H2 that describes the main idea of the content in the headings that follow.
+<<<<<<< HEAD
```

Comment:
Hmm sth went wrong here - looks like you were trying to resolve a conflict but the content got mixed up - could you take a look at the file preview and remove the unwanted elements? 

![image](https://user-images.githubusercontent.com/40655785/66305920-b526c700-e900-11e9-84ec-9aa6be3f6927.png)

Reply:
😠 How did this get into the doc :D We solved the conflict today and it looked good :D

Reply:
Looks good now :D

Reply:
Thank you both 😄